packages/ndla-ui/src/Breadcrumb/BreadcrumbBlock.tsx

```diff
@@ -6,26 +6,38 @@
  *
  */
 
-import React, { useRef, useLayoutEffect } from 'react';
-import PropTypes from 'prop-types';
-import BEMHelper from 'react-bem-helper';
-import { useComponentSize } from '@ndla/hooks';
+import React, { useRef } from 'react';
+import BEMHelper, { ReturnObject } from 'react-bem-helper';
+import { useComponentSize, useIsomorphicLayoutEffect } from '@ndla/hooks';
 import BreadcrumbItem from './BreadcrumbItem';
 
-const classes = BEMHelper({
+const classes: BEMHelper<ReturnObject> = BEMHelper({
   name: 'breadcrumb-block',
   prefix: 'c-',
 });
 
-const BreadcrumbBlock = ({ children, items }) => {
-  const olRef = useRef(null);
-  const containerRef = useRef(null);
+interface BreadcrumbItemI {
+  to: string;
+  name: string;
+}
+
+interface Props {
+  children: React.ReactNode;
+  items: BreadcrumbItemI[];
+}
+
+const BreadcrumbBlock: React.FunctionComponent<Props> = ({
```

Comment:
pr0 tip,  `FC` kan brukes istedenfor `FunctionComponent`, ikke så viktig nå, men for future references.

Reply:
Kan endre det om du (eller noen andre) har sterke meninger om det, men da syns jeg nesten vi burde oppdatere hele kodebasen.
Vi har mer `FunctionComponent` atm, så gikk for det siden jeg liker consistency mer enn korte typenavn. 

Jeg har egentlig ingen preferanse for noen av dem.

Reply:
Nei nei, har ingen sterke meninger, bare at jeg i din situasjon hadde gått for selve `FC` istedenfor `React.FunctionComponent`. Trodde ikke du visste om forkortelsen, men ser at du vet det og du har gjort et bevisst valg, så da er alt i orden.

Reply:
Har sett vi har brukt den tidligere, men så veldig bevisst annet enn "begge deler finnes jeg velger den jeg har sett mest" var det nok ikke :smile: 
Uansett så har vi brukt mer hjernekapasitet på å skrive disse kommentarene enn enn det noen av de (konsekvent eller ei) hadde spart oss, så tror jeg merger :joy: